subsys/testsuite/ztest/Kconfig

```diff
@@ -135,6 +135,12 @@ config ZTEST_SHUFFLE_TEST_REPEAT_COUNT
 
 endif #ZTEST_SHUFFLE
 
+config ZTEST_SUMMARY
+	bool "Display test summary"
+	default y
```

Comment:
I think for the overall config to make sense, there should be some dependency between this and `ZTEST_VERBOSE_SUMMARY`. As it stands, you could set `ZTEST_SUMMARY=n` and `ZTEST_VERBOSE_SUMMARY=y`, and that wouldn't make any sense.

However, it might be better to elaborate the existing config. Instead of having effectively 4 summary verbosity settings, one of which doesn't make any sense, turn `ZTEST_VERBOSE_SUMMARY` into a multi-choice with 3 settings.